examples/bukkit/automated-tests-shaded/pom.xml

```diff
@@ -0,0 +1,117 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>dev.jorel</groupId>
+	<artifactId>commandapi-automated-tests-shaded</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+	</properties>
+
+	<repositories>
+		<!-- Paper repository is required by MockBukkit -->
+		<repository>
+			<id>papermc</id>
+			<url>https://repo.papermc.io/repository/maven-public/</url>
+		</repository>
+	</repositories>
+
+	<dependencies>
+		<!-- MockBukkit for general testing utilities https://github.com/MockBukkit/MockBukkit -->
+		<!-- This has to go before spigot-api so MockBukkit's dependencies can have priority while testing -->
+		<!-- Otherwise, MockBukkit can get pretty cranky -->
+		<dependency>
+			<groupId>com.github.seeseemelk</groupId>
+			<artifactId>MockBukkit-v1.21</artifactId>
+			<version>3.133.2</version>
+			<scope>test</scope>
+		</dependency>
+
+		<!-- CommandAPI testing utilities -->
+		<!-- This has to go before the main CommandAPI dependency so the test-specific implementations get priority -->
+		<dependency>
+			<groupId>dev.jorel</groupId>
+			<artifactId>commandapi-bukkit-test-toolkit</artifactId>
+			<version>9.6.0-SNAPSHOT</version>
+			<scope>test</scope>
+		</dependency>
+
+		<!-- Main CommandAPI dependency for writing the plugin -->
+		<dependency>
+			<groupId>dev.jorel</groupId>
+			<artifactId>commandapi-bukkit-shade</artifactId>
+			<version>9.6.0-SNAPSHOT</version>
+			<scope>compile</scope>
+		</dependency>
+
+		<!-- Adds the Spigot API to write the plugin -->
+		<!-- If your plugin focuses on Paper, putting that dependency instead works as well -->
+		<dependency>
+			<groupId>org.spigotmc</groupId>
+			<artifactId>spigot-api</artifactId>
+			<version>1.21-R0.1-SNAPSHOT</version>
```

Comment:
I guess this could be the 1.21.1 dependency. Not a big deal but for consistency it would be nice.